vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -210,6 +210,33 @@ public void testVcfHtsExportSeveralStudies() throws Exception {
     }
 
     @Test
+    public void testVcfHtsExportOneFileFromOneStudyThatHasTwoFiles()
```

Comment:
We should not mention the HTS, it's a pure implementation detail. This comment applies to some other methods in this class.